src/Utils/Hexagons.svelte

```diff
@@ -21,39 +28,55 @@ let moveable;
-        binActuator = zero.repeat(96) + binActuator; //Pad with leading/trailing zeros to fill out to 128 'bits'
-    } else if (actuator >= 32 && actuator < 63) {
-        shiftActuator = Math.abs(1 << (actuator - 32)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = zero.repeat(64) + binActuator + zero.repeat(32);
-    } else if (actuator >= 64 && actuator < 95) {
-        shiftActuator = Math.abs(1 << (actuator - 64)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = zero.repeat(32) + binActuator + zero.repeat(64);
-    } else if (actuator >= 96 && actuator < 127) {
-        shiftActuator = Math.abs(1 << (actuator - 96)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = binActuator + zero.repeat(96);
+    for (let actuator of actuators) {
+        if (actuator <= 31) {
+            shiftActuator = Math.abs(1 << actuator); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(96) + binActuator; //Pad with leading/trailing zeros to fill out to 128 'bits'
+        } else if (actuator >= 32 && actuator < 63) {
+            shiftActuator = Math.abs(1 << (actuator - 32)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(64) + binActuator + zero.repeat(32);
+        } else if (actuator >= 64 && actuator < 95) {
+            shiftActuator = Math.abs(1 << (actuator - 64)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(32) + binActuator + zero.repeat(64);
+        } else if (actuator >= 96 && actuator < 127) {
+            shiftActuator = Math.abs(1 << (actuator - 96)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = binActuator + zero.repeat(96);
+        }
     }
     return [...Array(16).keys()].map(i => binActuator.slice(i * 8, (i+1) * 8)).reverse();
 }
 
 function buildCommandBlocks(active) {
     const [b0_0, b0_1, b0_2, b0_3, b1_0, b1_1, b1_2, b1_3, b2_0, b2_1, b2_2, b2_3, b3_0, b3_1, b3_2, b3_3] = getBytesForActuator(active);
-    block0_31.set([b0_0, b0_1, b0_2, b0_3]);
-    block32_63.set([b1_0, b1_1, b1_2, b1_3]);
-    block64_95.set([b2_0, b2_1, b2_2, b2_3]);
-    block96_127.set([b3_0, b3_1, b3_2, b3_3]);
+    block0_31.set([parseInt(b0_0,2), parseInt(b0_1,2), parseInt(b0_2,2), parseInt(b0_3,2)]);
+    block32_63.set([parseInt(b1_0,2), parseInt(b1_1,2), parseInt(b1_2,2), parseInt(b1_3,2)]);
+    block64_95.set([parseInt(b2_0,2), parseInt(b2_1,2), parseInt(b2_2,2), parseInt(b2_3,2)]);
+    block96_127.set([parseInt(b3_0,2), parseInt(b3_1,2), parseInt(b3_2,2), parseInt(b3_3,2)]);
```

Comment:
It would be good to change the getBytesForActuators to use integers instead of strings.